release-notes-3.3.0.md

```diff
@@ -1,8 +1,8 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
-#### New Features
+#### New Features  
+- [SCMOD-8484](https://portal.digitalsafe.net/browse/SCMOD-8484): Propagate failures through subtasks  
```

Comment:
Remove the link to JIRA as its an open source so only some people can access. Leave the JIRA number in place though.